application/app/Services/Domain/Admin/AdminUserManagementSearchService.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Services\Domain\Admin;
+
+use App\Http\Requests\Admin\AdminUserSearchRequest;
+use App\Models\AdminUser;
+use Illuminate\Contracts\Pagination\LengthAwarePaginator;
+
+/**
+ * 管理者ユーザー検索サービス
+ *
+ * @package App\Services\Domain
+ */
+class AdminUserManagementSearchService
```

Comment:
```suggestion
class AdminUserSearchService
```
Managementという単語は必要ですか？
冗長なだけな気がしますが